hummingbot/market/liquid/liquid_market.pyx

```diff
@@ -531,8 +531,10 @@ cdef class LiquidMarket(MarketBase):
                     min_order_size = math.pow(10, -rule.get(
                         "assets_precision", Constants.DEFAULT_ASSETS_PRECISION))
 
-                min_price_increment = math.pow(10, -rule.get(
-                    "quoting_precision", Constants.DEFAULT_QUOTING_PRECISION))
+                min_price_increment = product.get("tick_size")
+                if not min_price_increment:
```

Comment:
I have looked at the api response, some of `tick_size` are 0. Even though there are probably not-listed products, just to be sure, can you please add a check for 0 value here, ie. if it is 0 we'll use the default.

Reply:
Hi thanks for looking up this PR, I've made the change please do check it!